insolar/utils/utils.go

```diff
@@ -23,7 +23,7 @@ import (
 	"testing"
 
 	"github.com/pkg/errors"
-	"github.com/satori/go.uuid"
+	uuid "github.com/satori/go.uuid"
```

Comment:
uuid alias is not needed here (but is placed by idea)

Reply:
i've removed this alias, saved file and it appeared again :(